### ISSUE TITLE
Import RDM Files from  Remote Desktop Manger (Devolutions)

### ISSUE THREAD

**Issue by johnsie** [mRemoteNG/mRemoteNG#887]
Love your progam. It would be reallly good if it could import rdm files from the latest version of 'remote desktop manager' by devolutions. The file is an xml file should shouldn't be too hard to implement. I would rather use your product and support free software  give all my personal details to Devolution in order to get a closed source product.

**Comment by sparerd** [comment]
Yea should be fairly easy to do. I'll schedule for v1.77

**Comment by iaing80** [comment]
Hi,  any update on when V1.77 will be out?
Currenlty looking to switch to mRemoteNG howver unable to import my sessions.

**Comment by sparerd** [comment]
@iaing80 No, not yet. v1.76 has been delayed due to lack of development resources, though we are making a hard push to get it to a stable release very soon. I hope to be able to start doing v1.77 development work once v1.76 reaches beta.

**Comment by casao94** [comment]
@sparerd Would you be able to add this feature to v1.77? I didn't see this feature being worked on for v1.77

**Comment by sparerd** [comment]
@casao94 We haven't gotten to it yet, though it is still scheduled for that version

**Comment by drsnacek** [comment]
Hello, could you let me know what is the actual status of the implementation?

**Comment by anubis-genix** [comment]
I'm also trying to migrate from Devolutions RDM to mRemoteNG, but neither the .xml nor the .csv import works. Is there another way to get my database imported while the devs work on an official solution?

**Comment by KaledKelevra** [comment]
> I'm also trying to migrate from Devolutions RDM to mRemoteNG, but neither the .xml nor the .csv import works. Is there another way to get my database imported while the devs work on an official solution?

Same here... is there any update on this feature? I can't import RDM to mRemoteNG. :(

**Comment by ThinkPadNL** [comment]
Any suggestions on how to convert a Devolutions RDM export to a format that mRemoteNG understands would be great. I just spent an hour fiddling copy-pasting values from a .csv export from Devolutions RDM into an export from mRemoteNG and then importing again, but unfortunately without any luck :(

**Comment by vardars** [comment]
Hi, I want to work on this. Is it ok?

**Comment by DaveSchopp** [comment]
I'm hoping you've gotten this imported already, but here's what I ended up doing tonight.

I exported my entries to individual RDP files, them imported them into mRemoteNG. I then exported those entries into a CSV file and manually manipulated column data using a combination of Notepad++ and an application called  ["Rons Editor"](https://www.ronsplace.ca/Products/RonsEditor). Took me a couple hours for my 250 entries. After editing, I created a new connection file and imported the updated CSV file.

Having the ability to import would have saved me a ton of time!

**Comment by savornicesei** [comment]
@Kvarkas Looks like this issue can be moved to done due to merged PR#2208. Unless it needs to be documented (I'm not finding any mention of this in the v1.77 docs)?

**Comment by Kvarkas** [comment]
thanks to pointing me into that, will try to check
